### PR TITLE
DIV-5762 added prod divorce apps for cos cms

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

### JIRA link (if applicable) ###

DIV-5762

### Change description ###

* Updated AAT/Perftest configurations to use the latest java chart (and remove duplicates)
* Added prod configuration

**Please compare values with variables.tf, main.tf, aat.tfvars and prod.tfvars**

Related PR: https://github.com/hmcts/div-case-orchestration-service/pull/484

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/common/divorce/div-cos.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-cos
+    version: 1.3.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "baaed5d0-7666-4dce-bf74-edfceeb566f9"
+      image: hmctspublic.azurecr.io/div/cos:prod-aed712be
```
Comment (hosainnet) [inline k8s/prod/common/divorce/div-cos.yaml]: To do: update this once https://github.com/hmcts/div-case-orchestration-service/pull/484 is merged

File: k8s/perftest/common/divorce/div-cms.yaml [hunk 2]
```diff
@@ -27,29 +27,12 @@ spec:
       readinessPeriod: 15
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
-      image: hmctspublic.azurecr.io/div/cms:prod-d2b80c6c
-      ingressHost: div-cms-perftest.service.core-compute-perftest.internal
+      image: hmctspublic.azurecr.io/div/cms:prod-33c59844
```
Comment (hosainnet) [inline k8s/perftest/common/divorce/div-cms.yaml]: To do: update this once https://github.com/hmcts/div-case-orchestration-service/pull/484 is merged

File: k8s/aat/common/divorce/div-cos.yaml [hunk 3]
```diff
@@ -28,54 +28,16 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
       image: hmctspublic.azurecr.io/div/cos:prod-135ec6cc
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-cos.yaml]: To do: update this once https://github.com/hmcts/div-case-orchestration-service/pull/484 is merged

File: k8s/perftest/common/divorce/div-cos.yaml [hunk 4]
```diff
@@ -28,54 +28,16 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
       image: hmctspublic.azurecr.io/div/cos:prod-135ec6cc
```
Comment (hosainnet) [inline k8s/perftest/common/divorce/div-cos.yaml]: To do: update this git hash once https://github.com/hmcts/div-case-orchestration-service/pull/484 is merged

File: k8s/perftest/common/divorce/div-cos.yaml [hunk 5]
```diff
@@ -28,54 +28,16 @@ spec:
...
-        DIV_SCHEDULER_DB_NAME: "div_scheduler"
-        DIV_SCHEDULER_DB_USER_NAME: "div_scheduler@div-cos-perftest"
-        FLYWAY_URL: "jdbc:postgresql://div-cos-perftest.postgres.database.azure.com:5432/div_scheduler"
-        SCHEDULER_SCHEDULES_CREATE_BULK_CASES_ENABLED: false
-        SCHEDULER_RE_CREATE: true
-        SCHEDULER_ENABLED: true
         FEATURE_RESP_SOLICITOR_DETAILS: "true"
+        SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE: false
+        SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_ENABLED: true
+        DATAEXTRACTION_STATUS_DA_EMAILTO: da_data_extraction@sharklasers.com
```
Comment (jvictorcf) [inline k8s/perftest/common/divorce/div-cos.yaml]: Does this environment use test data?

Comment (hosainnet) [reply k8s/perftest/common/divorce/div-cos.yaml]: Yes, it's prod like but fake data

Comment (jvictorcf) [reply k8s/perftest/common/divorce/div-cos.yaml]: OK. This data will end up in a public e-mail inbox.
It doesn't sound like an issue, but I'm just letting you know.

File: k8s/aat/common/divorce/div-cms.yaml [hunk 6]
```diff
@@ -14,7 +14,7 @@ spec:
   chart:
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-cms.yaml]: https://github.com/hmcts/div-case-maintenance-service/blob/master/infrastructure/variables.tf
https://github.com/hmcts/div-case-maintenance-service/blob/master/infrastructure/main.tf
https://github.com/hmcts/div-case-maintenance-service/blob/master/infrastructure/aat.tfvars

File: k8s/aat/common/divorce/div-cos.yaml [hunk 7]
```diff
@@ -14,7 +14,7 @@ spec:
   chart:
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-cos.yaml]: https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/variables.tf
https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/main.tf
https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/aat.tfvars

File: k8s/prod/common/divorce/div-cms.yaml [hunk 8]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (hosainnet) [inline k8s/prod/common/divorce/div-cms.yaml]: https://github.com/hmcts/div-case-maintenance-service/blob/master/infrastructure/main.tf
https://github.com/hmcts/div-case-maintenance-service/blob/master/infrastructure/variables.tf
https://github.com/hmcts/div-case-maintenance-service/blob/master/infrastructure/prod.tfvars

File: k8s/prod/common/divorce/div-cos.yaml [hunk 9]
```diff
@@ -0,0 +1,39 @@
+---
```
Comment (hosainnet) [inline k8s/prod/common/divorce/div-cos.yaml]: https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/main.tf
https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/variables.tf
https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/prod.tfvars

**Pull request merged**